.circleci/config.yml

```diff
@@ -40,3 +43,37 @@ jobs:
           | python3 - --version 1.1.0"
       - run: poetry build
       - run: POETRY_HTTP_BASIC_PYPI_PASSWORD=${PYPI_PASSWORD} poetry publish
+  test_installation:
+    machine:
+      image: ubuntu-2004:202010-01
+    steps:
+      - run: sudo apt-get update
+      - run: sudo apt-get install python3 python3-dev curl git -y
+      - checkout
+      - run: "curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py
+          | python3 - --version 1.1.0"
```

Comment:
Could you make this use version "1.1.4"? It probably won't change anything for this test, but we're working on using that version everywhere and it fixes some issues.